pkgs/applications/editors/vim/plugins/overrides.nix

```
@@ -1006,6 +1010,8 @@
     meta.maintainers = with lib.maintainers; [ vcunat ];
   };
 
+  middleclass = neovimUtils.buildNeovimPlugin { luaAttr = "middleclass"; };
```

How about adding a test?

```suggestion
  middleclass = neovimUtils.buildNeovimPlugin { luaAttr = "middleclass"; nvimRequireCheck = "middleclass"; };
```

---

Thanks for letting me know this exists!